src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -147,9 +147,12 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
         List<FeedbackSessionAttributes> sessions =
                 getFeedbackSessionsForCourse(courseId);
         List<FeedbackSessionAttributes> viewableSessions = new ArrayList<FeedbackSessionAttributes>();
-        for (FeedbackSessionAttributes session : sessions) {
-            if (isFeedbackSessionViewableTo(session, userEmail)) {
-                viewableSessions.add(session);
+        if (!sessions.isEmpty()) {
+            InstructorAttributes instructor = instructorsLogic.getInstructorForEmail(courseId, userEmail);
```

Comment:
Can the instructor be null? The method does not require the userEmail to belong to an instructor right?